dracut/30disk-uuid/disk-uuid.service

```diff
@@ -1,18 +1,17 @@
 [Unit]
-Description=Generate new UUID for disk GPT %I
+Description=Generate new UUID for disk GPT if necessary
 DefaultDependencies=no
+
 Wants=local-fs-pre.target
 Before=local-fs-pre.target systemd-fsck-root.service
-Wants=systemd-udevd.service
-After=systemd-udevd.service
-Requires=%i.device
-After=%i.device
+
+Wants=systemd-udevd.service dev-disk-by\x2dlabel-ROOT.device
```

Comment:
The root device can be recreated by Ignition and is not really required to exist. I suggest waiting for one of the USR partitions.

Reply:
What do you mean is not required to exist? How can ROOT not exist? I guess I can try with USR-A, but given how much time I already wasted on this, I won't try until alpha test are working again.

Reply:
The root partition can be created in Ignition and consist of a RAID or be on a different drive.